pkg/registry/registry.go

```diff
@@ -30,6 +30,14 @@ const (
 	IANAReversedEnterpriseID uint32 = 29305
 )
 
+// enum for flowType field in Antrea registry.
+const (
+	IntraNode    = uint8(1)
+	InterNode    = uint8(2)
+	ToExternal   = uint8(3)
+	FromExternal = uint(4)
```

Comment:
Why `FromExternal` type is different from others?

Reply:
It was a typo. Fixed.